### PR TITLE
Adds try/catch wrappers for routes

### PR DESCRIPTION
**Pull request by eahefnawy**

### CODE REVIEW

File: examples/default/index.js [hunk 1]
```diff
@@ -11,73 +11,85 @@
...
-  let result = await getTodos(req.query.status, req.query.meta ? true : {});
-
-  // Return the results
-  res.send({
-    items: result.items
-  })
-})
+ */
+api.get("/todos", async (req, res, next) => {
+  try {
```
Comment (dougmoscrop) [inline examples/default/index.js:18]: These aren't necessary, as the runtime does it for you (currently via express-async-errors)

Comment (eahefnawy) [reply examples/default/index.js:18]: I've found them necessary when building apps for the cloud hackathon. Errors were not handled without them 🤔

Comment (dougmoscrop) [reply examples/default/index.js:18]: I think that should be classified as a bug then!

Comment (eahefnawy) [reply examples/default/index.js:18]: hmm ok, feel free to close this PR then 👌

Comment (jeremydaly) [reply examples/default/index.js:18]: I was thinking the same thing. They should be handled for you, and if you have error middleware, you can control the output.

**Comment by dougmoscrop** [comment]
I tested this and confirmed it behaves as expected/desired, so closing.

**Pull request closed without merging**